sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java

```diff
@@ -52,24 +52,105 @@ private Constants() {
      */
     public static final class ConnectionStringConstants {
         /**
-         * The AccountName key.
+         * The setting name for the storage account name.
          */
-        public static final String ACCOUNT_NAME = "accountname";
+        public static final String ACCOUNT_NAME_NAME = "AccountName";
```

Comment:
ACCOUNT_NAME_NAME? Why not ACCOUNT_NAME as it was?

Reply:
agree, will revert this change.